app/dao/organisations_dao.py

```diff
@@ -0,0 +1,21 @@
+from app import db
+from app.dao.dao_utils import transactional
+from app.models import Organisation
+
+
+def dao_get_organisations():
+    return Organisation.query.all()
+
+
+def dao_get_organisation_by_id(org_id):
+    return Organisation.query.filter_by(id=org_id).one()
+
+
+@transactional
+def dao_create_organisation(organisation):
```

Comment:
Hmm, this should take some kwargs like other dao methods do. Otherwise it's just doing the same as `dao_update_organisation` when it probably should be doing more.